tensorflow_datasets/core/download/kaggle_test.py

```diff
@@ -32,14 +32,18 @@ class KaggleTest(testing.TestCase):
   def test_competition_download(self):
     filenames = ["a", "b"]
     with testing.mock_kaggle_api(filenames):
-      downloader = kaggle.KaggleCompetitionDownloader("digit-recognizer")
+      competition_name = "digit-recognizer"
+      downloader = kaggle.KaggleCompetitionDownloader(competition_name)
       self.assertEqual(downloader.competition_files, ["a", "b"])
       with testing.tmp_dir() as tmp_dir:
         for fname in downloader.competition_files:
           out_path = downloader.download_file(fname, tmp_dir)
           self.assertEqual(out_path, os.path.join(tmp_dir, fname))
           with tf.io.gfile.GFile(out_path) as f:
             self.assertEqual(fname, f.read())
+      with testing.tmp_dir():
+        downloader.download_all_files()
+        self.assertCountEqual(os.listdir(competition_name), downloader.competition_files)
```

Comment:
Kokoro fails on py2 and tf12.

Could you use six.assertCountEqual?

Reply:
Thank you resolved!